### PR TITLE
Inject Airflow environment variables in docker-compose config

### PR DESCRIPTION
**Pull request by mjgiarlo**
Connects to #287

This commit allows the running airflow services to be configured dynamically via environment variables.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ ENV SLUGIFY_USES_TEXT_UNIDECODE "yes"
 
 USER airflow
 
-COPY requirements.txt pyproject.toml poetry.lock .
+COPY requirements.txt pyproject.toml poetry.lock ./
```
Comment (lwrubel) [inline Dockerfile:12]: When using COPY with more than one source file, Docker expects the destination to be a directory and end with a `/`

**Comment by lwrubel** [comment]
Puppet PR to accompany this: https://github.com/sul-dlss/puppet/pull/9411/files (will take it out of draft once this PR is reviewed to confirm it's the right set of variables).

All of the variables are in vault for dev. Prod and stage will need the new ones.

**Review by jermnelson — CHANGES_REQUESTED** [review]
Looks good overall. Just a request to update the README at https://github.com/sul-dlss/libsys-airflow#running-locally-with-docker to add these variables to the local `.env` file

**Comment by lwrubel** [comment]
Thanks, that's a good suggestion. Instructions added.

**Review by jermnelson — APPROVED** [review]

**Pull request merged**